ONPREM-1276 | Added rabbitmq instruction in Server-4.2 (#8820)
circleci/circleci-docs: jekyll/_cci2/server/v4.2/installation/upgrade-server.adoc

- xref:../installation/phase-2-core-services#api-token[api-token]
- xref:../installation/phase-2-core-services#session-cookie[session-cookie]

> - xref:../installation/phase-2-core-services#rabbitmq-configurations-and-auth-secrets[rabbitmq-key]

Author: We dont need this.

. Check the link:https://circleci.com/server/changelog/[changelog] and make sure there are no actions you need to take before deploying a new version.

> . Export the "RabbitMQ" password as we are upgrading helm chart for the same as well.

Reviewer: @atulsingh0 does this mean We are upgrading the helm chart for RabbitMQ? Not 100% sure what is meant by "the same" here that's all

Author: In this release, we are updating the RabbitMQ chart as well

> . Export the "RabbitMQ" password as we are upgrading helm chart for the same as well.
> 
> [source,shell]
> ----

Reviewer: ```suggestion
. Export the "RabbitMQ" password. We are also upgrading the RabbitMQ Helm chart during this upgrade.
+
[source,shell]
----
```
